shared/src/main/kotlin/com/egm/stellio/shared/model/NgsiLdDataRepresentation.kt

```diff
@@ -28,9 +29,14 @@ data class NgsiLdDataRepresentation(
             acceptMediaType: MediaType
         ): NgsiLdDataRepresentation {
             val optionsParam = queryParams.getOrDefault(QueryParameter.OPTIONS.key, emptyList())
+            val formatParam = queryParams.getOrDefault(QueryParameter.FORMAT.key, emptyList())
+            val attributeRepresentation = when {
```

Comment:
since you are in it and since it is very related, also add support for the `simplified` value for the `format` query param (synonym of `keyValues`)